launcher/ui/pages/global/APIPage.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
```

Comment:
Can you open the .ui file once with Qt Designer and close it again, so it can figure out formatting and ordering and stuff?